client/signup/steps/domains/test/utils.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @jest-environment jsdom
+ */
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	getExternalBackUrl,
+	backUrlExternalSourceStepsOverrides,
+	backUrlSourceOverrides,
+} from '../utils';
+
+const backUrlSourceOverrideKeys = Object.keys( backUrlSourceOverrides );
+const backUrlSourceOverrideKey = backUrlSourceOverrideKeys[ 0 ];
+
+describe( 'getExternalBackUrl', () => {
+	test( 'it should return false if given no first source arg', () => {
+		const result = getExternalBackUrl( undefined );
+		expect( result ).toBe( false );
+		expect( result ).toBeFalsy();
```

Comment:
`getExternalBackUrl()` explicitly returns `false` so why do we have to check if the result is `falsy` as well?